core/src/main/java/bio/terra/pearl/core/dao/kit/KitRequestDao.java

```diff
@@ -28,8 +28,8 @@ public List<KitRequest> findByEnrollee(UUID enrolleeId) {
         return super.findAllByProperty("enrollee_id", enrolleeId);
     }
 
-    public Optional<KitRequest> findByEnrolleeAndBarcode(UUID enrolleeId, String barcode) {
-        return findByTwoProperties("enrollee_id", enrolleeId, "kit_barcode", barcode);
+    public Optional<KitRequest> findByEnrolleeAndLabel(UUID enrolleeId, String kitLabel) {
+        return findByTwoProperties("enrollee_id", enrolleeId, "kit_label", kitLabel);
```

Comment:
purely curious: why is `label` preferred over `barcode`?

Reply:
DSM and GP call it a label. IMO barcode makes a tiny bit more sense, but i'd rather we all speak the same language given the complexity of the integration